src/seedu/addressbook/commands/Command.java

```diff
@@ -40,7 +40,7 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
      * Executes the command and returns the result.
      */
     public CommandResult execute(){
-        throw new UnsupportedOperationException("This method should be implement in child classes");
+        throw new UnsupportedOperationException("This method should be implemented in child classes");
```

Comment:
Looks like you mixed some commits from a different LO here.
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.